README.md

```diff
@@ -0,0 +1,45 @@
+# Schul-Cloud for Android
+
+[![Build Status](https://travis-ci.org/schul-cloud/schulcloud-mobile-android.svg?branch=dev)](https://travis-ci.org/schul-cloud/schulcloud-mobile-android)
+[![GitHub version](https://img.shields.io/github/release/schul-cloud/schulcloud-mobile-android.svg?colorB=4c1)](https://github.com/schul-cloud/schulcloud-mobile-android/releases)
+[![Issues: Jira](https://img.shields.io/badge/Issues-Jira-blue.svg)](https://ticketsystem.schul-cloud.org/projects/AN/issues)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-blue.svg)](http://makeapullrequest.com)
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](https://www.firsttimersonly.com/)
+
+
+[![Semver](http://img.shields.io/SemVer/2.0.0.png)](http://semver.org/spec/v2.0.0.html)
+[![Apache 2.0 license](https://img.shields.io/badge/License-Apache%202.0-green.svg?colorB=4c1)](http://www.apache.org/licenses/LICENSE-2.0)
+[![Awesome Badges](https://img.shields.io/badge/badges-awesome-green.svg?colorB=4c1)](https://github.com/Naereen/badges)
+
+
+Native Android application for [Schul-Cloud](https://schul-cloud.org/).
+
+
+## Issues
+
+The [issues for this project](https://ticketsystem.schul-cloud.org/projects/AN/issues) are managed in the global ticket system for Schul-Cloud.
+
+
+## How to get started
+
+Thanks for your interest in contributing! There are many ways to contribute to this project. Get started [here][contributing].
+
+
+## Contributing
```

Comment:
You can removes this, moved to own file.

Reply:
Do you intend to remove the whole "How to get started" section? I'm not sure whether everyone will see the contributing doc otherwise.

Reply:
I only refer to the `Contributing` section. It is empty and the link is provided in the `How to get started` section.